File: Data Structures/Arrays/C++/04_Left Rotation.cpp
// Date - Oct 5, 2020 By Jatin Sharma

#include <iostream>
using namespace std;

int main()
{
    int n, k;
    cin >> n >> k;
    int a[n];
    
    for (int i = 0; i < n; i++)
        cin >> a[i];

    int j = k % n;
    
    for (int i = j; i < n; i++)
        cout << a[i] << " ";
    
    for (int i = 0; i < j; i++)
        cout << a[i] << " ";
    return 0;
}

File: Data Structures/Arrays/Python/04_Left Rotation.py
# Date - Oct 5, 2020 By Jatin Sharma


if __name__ == '__main__':
    nd = input().split()

    n = int(nd[0])

    d = int(nd[1])

    a = list(map(int, input().rstrip().split()))
    '''for i in range(0,d,1):
        temp = a[0]
        for j in range(0,n-1,1):
            a[j] = a[j+1]
        a[n-1] = temp

    for x in range(n):
        print(a[x] , end = " ")'''
    for y in range(d, n, 1):
        print(a[y], end=' ')
    for z in range(0, d, 1):
        print(a[z], end=' ')
